rs21io/mothr-component-library
# Mothrmobil 784 dropdown 2

https://rs21inc.atlassian.net/browse/MOTHRMOBIL-784

**Changelog**

- Configuration for Storybook + Vuetify + Dart-sass (scss) 
- Added Dropdown story that uses `v-select`

**Steps for manual testing**

- `npm run storybook` and confirm that vuetify components (button and dropdown) are rendered correctly with controls and docs 

**Checklist**

- [x] I have merged the `main` branch into my branch directly prior to opening this PR
- [ ] I have updated the README + relevant Guru or Confluence docs
- [x] The linter is not throwing errors or warnings
- [x] All new and existing unit tests are passing locally
- [x] All new and existing integration tests are passing locally
- [x] Updated the app version

https://www.loom.com/share/0a8e6d5639744c1ba71d862ad2ca4079
